### PR TITLE
Read output from Arduino

### LINKED ISSUE #22: Investigate how to use Celery for async quering of Arduino

### PR DESCRIPTION
**Pull request by sondr3**
This closes #22.

### CODE REVIEW

**Comment by sondr3** [comment]
Some notes and thoughts from the current prototype:

- I switched back and forth between RabbitMQ and Redis during the last trial, and it seems that RabbitMQ works better, when quitting Redis for instance you get a wall of errors on my machine which I didn't like.
- I removed the `sensors.tasks.read()` function because I realized we don't actually need it
- We have a problem with the current always being read as above 2000 for some reason, I'll open issues about this
- WE HAVE A WORKING PROTOTYPE 🎉 

It will automatically run the `insert_coffee()` task and add the result to the database, woop woop.

File: README.md [hunk 1]
```diff
@@ -52,7 +53,44 @@ that everything is up to date, run the following commands:
...
+is our broker, then to actually run Celery and RabbitMQ you need to run the
+following two commands in two separate terminal windows:
+
+1. `rabbitmq-server`
+2. `celery -A moccapi worker -l info -B`
+
+Running the commands from Python you need to start the Python Console from
+Pycharm via `Tools -> Python Console`, then run the following:
+
+``` python
```
Comment (sondr3) [inline README.md]: Reminder to myself to remove this section as it doesn't exist in the code anymore :man_facepalming:

**Review by Eivinddh — DISMISSED** [review]
Looking good partner

**Review by Eivinddh — APPROVED** [review]
Tried and tested

**Pull request merged**